exporter/postgres/setup_pg10.sql

```diff
@@ -10,6 +10,29 @@ GRANT pg_monitor to ccp_monitoring;
 
 CREATE SCHEMA IF NOT EXISTS monitor AUTHORIZATION ccp_monitoring;
 
--- There are currently no SECURITY DEFINER functions required as long as the "pg_monitor" role is granted to ccp_monitoring.
+CREATE OR REPLACE FUNCTION monitor.pgbackrest_info() returns jsonb 
+    LANGUAGE plpgsql SECURITY DEFINER
+AS $$
+DECLARE
+    data jsonb;
+BEGIN
+    -- Get pgBackRest info in JSON format
+
+    -- Create a temp table to hold the JSON data
+    CREATE TEMP TABLE temp_pgbackrest_data (data jsonb);
+
+    -- Copy data into the table directory from the pgBackRest into command
+    COPY temp_pgbackrest_data (data)
+        FROM program
+            'pgbackrest --output=json info | tr ''\n'' '' ''' (format text);
```

Comment:
This is the function provided from the pgbackrest docs in order to get the current backup status in a parsable format that postgres can read.

https://pgbackrest.org/user-guide.html#monitor

Not sure how the --config option would help there since that's just used to designate the config file being used when pgbackrest is run.

Reply:
config file hold repo info, which inturn read by pgbackrest to generate info output

Reply:
That's going to make things a lot more complex for configuring alerts. Will have to think on that and discuss further.